tests/NHSD.BuyingCatalogue.Testing.Data/Entities/SolutionEntity.cs

```diff
@@ -96,7 +124,12 @@ public static async Task<IEnumerable<SolutionEntity>> FetchAllAsync()
 
         public static async Task<SolutionEntity> GetByIdAsync(string solutionId)
         {
-            return (await FetchAllAsync()).First(item => solutionId == item.Id);
+            return (await FetchAllAsync()).First(item => solutionId == item.SolutionId);
+        }
+
+        public async Task InsertAndSetCurrentForSolutionAsync()
+        {
+            await InsertAsync();
         }
```

Comment:
We should be able to remove this following the changes requested in earlier comments.

Reply:
Method `InsertAndSetCurrentForSolutionAsync` removed, and all references are updated to `InsertAsync` method.